src/index.html

```diff
@@ -1,13 +1,42 @@
+<!-- #region head -->
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      alt="iMac"
+      class="card__img">
```

Comment:
You need to check and fix it everywhere 
<img width="1052" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/70abe84a-78a9-47e0-b4b3-0c37d9567cc7">
```suggestion
      <img
        src="images/imac.jpeg"
        alt="iMac"
        class="card__img"
      >
```